src/Win32/Win32Helper.cs

```diff
@@ -115,5 +115,14 @@ internal bool IsWindowsRS5OrLater()
         {
             return IsAtLeastWin10WithSpecificBuild(17713); // Build 17713 is confirmed in the RS5 range
         }
+
+        /// <summary>
+        /// Check whether current OS is Win11 (Sun Valley) or later
+        /// </summary>
+        /// <returns>True if and only if the OS is at least Win11</returns>
+        internal bool IsWindowsSVOrLater()
```

Comment:
Suggested name: `IsWindows11OrLater`

Reply:
This diverges from the style of other method names in this class.